qml/pages/SettingsPage.qml

```diff
@@ -147,13 +149,25 @@ Dialog {
 
             SectionHeader{ text: qsTr("Notifications") }
 
-            IconTextSwitch {
-                id: notifications
-                text: qsTr("Enable notifications")
-                icon.source: "qrc:///images/icon-notifications.png"
-                icon.scale: Theme.iconSizeMedium/icon.width // Scale icons according to the screen sizes
-                checked: settings.enableNotifications
-                description: qsTr("%1 will send you notifications when you have a new message, a new notification or a friend request.").arg("Sailbook")
+            TextSwitch {
+                id: notifyRequests
+                text: qsTr("Notify friend requests")
+                checked: settings.notifyRequests
+                description: qsTr("%1 will send you notifications when getting a friend request.").arg("Sailbook")
+            }
+
+            TextSwitch {
+                id: notifyMessages
+                text: qsTr("Notify incoming messages")
+                checked: settings.notifyMessages
+                description: qsTr("%1 will send you notifications when receiving a new message.").arg("Sailbook")
+            }
+
+            TextSwitch {
+                id: notifyNotifications
+                text: qsTr("Notify new notifications")
+                checked: settings.notifyNotifications
+                description: qsTr("%1 will send you notifications when you have a new Facebook notification.").arg("Sailbook")
```

Comment:
Remove "Facebook" here please to be consistent again with the 2 above.

Reply:
I have no strong feelings about this, but doesn't it sound a little stupid to say "will send you a notification when you have a new notification"? Having twice the same term in the same sentence is a little confusing...

If you are sure, I'll re-spin the patch :) Thanks for your quick review!

Reply:
Oh you're right :)
Maybe I need to change the 2 above one to "Facebook friend request" and "Facebook message"?